core/src/main/scala/org/apache/spark/FutureAction.scala

```diff
@@ -89,6 +89,14 @@ trait FutureAction[T] extends Future[T] {
    */
   override def value: Option[Try[T]]
 
+  // These two methods must be implemented in Scala 2.12, but won't be used by Spark
+
+  def transform[S](f: (Try[T]) => Try[S])(implicit executor: ExecutionContext): Future[S] =
```

Comment:
I tried compiling a small app that calls `RDD.countAsync` (which returns a `FutureAction`) and even implements a custom `FutureAction` and compiled vs 2.2.0, then ran vs this build, and it worked. I believe this may be legitimately excluded from MiMa.